.github/workflows/release.yml

```
@@ -121,14 +118,29 @@
         echo "registry=https://registry.npmjs.org/" >> ~/.npmrc
 
     - name: Publish to npm
+      if: github.event_name != 'pull_request'
       run: npm publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTOMATION_TOKEN }}
 
     - name: Upload Release Assets
+      if: github.event_name != 'pull_request'
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)